lib/tasks/configuration.rake

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_relative 'task_helpers'
+
+namespace :configuration do
+  desc 'Create an administrator account'
+  task check: :environment do
+    required_env_vars = %w[SECRET_KEY_BASE BIGBLUEBUTTON_ENDPOINT BIGBLUEBUTTON_SECRET DATABASE_URL REDIS_URL].freeze
+
+    # Initial check that variables are set
+    info 'Checking required environment variables:'
+    required_env_vars.each do |var|
+      failed("#{var} not set correctly") if ENV[var].blank?
+    end
+    passed
+
+    info 'Checking connection to Postgres Database:'
+    begin
+      ActiveRecord::Base.establish_connection # Establishes connection
+      ActiveRecord::Base.connection # Calls connection object
+      failed('Failed to connect to Database') unless ActiveRecord::Base.connected?
```

Comment:
Just for clarity `failed` already have a `Failed - ` prefix so this will be `Failed - Failed to connect to Database`.
I think something like `Failed - Unable to connect to Database` may be better.